packages/contracts/contracts/interfaces/v0.8.x/ISplitAtomicFactoryV0.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Creatd By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+import {Split} from "./ISplitAtomicV0.sol";
+
+interface ISplitAtomicFactoryV0 {
+    /**
+     * @notice This contract was deployed.
+     * @param implementation address with the implementation of the contract
+     * @param type_ type of this contract
+     */
+    event Deployed(address indexed implementation, bytes32 indexed type_);
+    /**
+     * @notice New split atomic contract was created.
+     * @param splitAtomic address of the newly created split atomic contract
+     */
+    event SplitAtomicCreated(address indexed splitAtomic);
+
+    /**
+     * @notice Initializes the contract with the provided `splits`.
+     * Only callable once.
+     * @param splits Splits to configure the contract with. Must add up to
+     * 10_000 BPS.
+     * @return splitAtomic The address of the newly created split atomic
+     * contract
+     */
+    function createSplit(
+        Split[] calldata splits
+    ) external returns (address splitAtomic);
```

Comment:
```suggestion
    ) external returns (address splitAtomic, uint256 intendedProjectIdMetadata);
    // emit event connecting the new split and intended project ID
```

Reply:
@Jbern16 @lindsgil I think we ended up handling this with frontend solution (localstorage), which I like much more since this is a permissionless call and subject to spamming. going to go ahead and close this one.